src/python/dxpy/templating/templates/nextflow/src/nextflow.sh

```diff
@@ -442,53 +448,58 @@ main() {
     exit $ret
 }
 
-# On exit, for the Nextflow task sub-jobs
-nf_task_exit() {
-  ret=$?
-  if [ -f .command.log ]; then
-    dx upload .command.log --path "${cmd_log_file}" --brief --wait --no-progress || true
-  else
-    >&2 echo "Missing Nextflow .command.log file"
-  fi
-  # mark the job as successful in any case, real task
-  # error code is managed by nextflow via .exitcode file
-  if [ -z ${exit_code} ]; then export exit_code=0; fi
-
-  # Make sure that subjob with errorStrategy == terminate end in 'failed' state
-  wait_time=240
+wait_for_terminate_or_retry() {
```

Comment:
Refactor -- separate wait_for_terminate_or_retry from nf_task_exit

Reply:
Only look for the error strategy files in the workspace _if_ in the condition where the subjob had error